### PR TITLE
Fix links to visa info and grant application form

### PR DESCRIPTION
**Pull request by rebkwok**

### CODE REVIEW

File: _pages/visas.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ title: Visa Information
 If you’re planning to attend DjangoCon Europe, first check whether you need a visa to 
 visit the UK. Fill in the form on [this page](https://www.gov.uk/check-uk-visa/y) to check 
 whether you need a visa.  A standard UK visitor visa costs £100, and may take around 6 weeks to process.  You can find more information on visa requirements for visiting the UK 
-here.
+[here](https://www.gov.uk/entering-staying-uk/visit-and-transit-visas).
```
Comment (thibaudcolas) [inline _pages/visas.md:15]: Same as above.

```suggestion
on the GOV.UK [Visit and transit visas](https://www.gov.uk/entering-staying-uk/visit-and-transit-visas) page.
```

File: _pages/opportunity-grants.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ You can apply for 3 types of grant:
 - Visa application fee: reimbursement for the cost of a UK Standard Visitor visa (up to £100).
 
 {% comment %}
-To apply, please fill out [this form](https://2023.djangocon.eu/opportunity-grants/).
+To apply, please fill out [this form]({{ site.opportunity_grant_application }}).
```
Comment (thibaudcolas) [inline _pages/opportunity-grants.md:61]: I’d prefer we use link text descriptive of the target page so screen reader users can navigate link-by-link, doesn’t have to be this exact text if you’d rather use something else:

```suggestion
To apply, please fill out our [Opportunity Grant Application]({{ site.opportunity_grant_application }}) form.
```

File: _pages/opportunity-grants.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ You can apply for 3 types of grant:
 - Visa application fee: reimbursement for the cost of a UK Standard Visitor visa (up to £100).
 
 {% comment %}
```
Comment (thibaudcolas) [inline _pages/opportunity-grants.md:60]: Double checking whether it’s intentional this is still commented out?

Comment (rebkwok) [reply _pages/opportunity-grants.md:60]: Yes, that's correct, I don't want to open the grant application until the form is reviewed by everyone and we're happy about the info we're collecting

**Review by thibaudcolas — APPROVED** [review]
Update link texts

**Pull request merged**